info/guide/guide-chapter-3.md

```diff
@@ -890,7 +891,7 @@ inner
 out
 enum / annotation
 companion
-inline / noinline
+value / inline / noinline
```

Comment:
It's not from 1.6, we don't even support 1.6 in diktat now. Please check if it appeared in 1.4 or 1.5

Reply:
1.5

Reply:
Fixed all